tests/gui_tests/test_gui_methods.py

```diff
@@ -18,22 +20,49 @@
 # for most of the tests without -s, and then the GUI tests with -s.
 
 
+def get_test_datastore():
+    if os.path.exists("test_gui.db"):
```

Comment:
We use the "test_gui.db" string 4 times. It would be my preference to declare it once as a constant, then use that constant 4 times.

Reply:
Fixed in 19340c5d